website/docs/docs/dbt-versions/release-notes/04-Sept-2023/removing-prerelease-versions.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Removing old (prerelease) versions of dbt from dbt Cloud when (latest) is available"
+description: "Improving the version selection options by removing prerelease versions whenever the latest version is available."
+sidebar_label: "Removing old prerelease versions from dbt Cloud"
```

Comment:
these three YAML keys all should include a prefix. 

check out the process page https://www.notion.so/dbtlabs/Request-a-dbt-Cloud-release-note-f962f39b49af4a23811d70984fef4f65?pvs=4 and previous release notes for deets